### PR TITLE
Workbox Precaching Updates & Auto Updated On Task

### PR DESCRIPTION
**Pull request by gauntface**
@petele please check out package.json and gulp-tasks/auto-update-on.js. This will auto update the wf_updated_on tag of any changed files to the current date on precommit.

@jeffposnick could you give a once over of the content.

### CODE REVIEW

**Comment by petele** [comment]
@gauntface - how hard would it be to separate the two changes into separate PRs? one for `wf_updated_on` and one for the content?

**Comment by gauntface** [comment]
@petele removed from pr

**Comment by petele** [comment]
Awesome, thanks! Can't wait for it to land, looks awesome.

**Comment by gauntface** [comment]
@jeffposnick That needs writing and issue is (https://github.com/GoogleChrome/workbox/issues/1208)

This is largely geared towards the getting started flow of - add CDN and register some run time routes / strategies, then add precaching.

The generation flow will be in the guides section in top level.

File: src/content/en/tools/workbox/guides/precache-files/cli.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  },
+
+  ...
+
+]);</code></pre>
+
+When you make a change to your project, run the inject manifest command
+and you'll have an up to date service worker with precache support.
+
+{% include web/tools/workbox/guides/_shared/precache-config.md %}
```
Comment (petele) [inline src/content/en/tools/workbox/guides/precache-files/cli.md]: When deploying this morning, I learned these have to be fully quoted. I've fixed in 24afe4d, and updated the tests to verify in the future (#5690)

**Pull request merged**